libdnf/sack/query.cpp

```diff
@@ -552,6 +581,20 @@ class Query::Impl {
     std::unique_ptr<PackageSet> result;
     std::vector<Filter> filters;
     void apply();
+
+    /**
+    * @brief It accepts stings of whole nevras and apply them to the query. It requires full
+    * nevra without globs.
+    * For dnf-2.8.9-1.fc27.noarch it accepts dnf-0:2.8.9-1.fc27.noarch or dnf-2.8.9-1.fc27.noarch
+    * But for package gedit-3:3.22.1-2.fc27.x86_64 the string gedit-3.22.1-2.fc27.x86_64 is
+    * incorrect and there will be no result for the query.
+    *
+    * Allowed compare types - only HY_EQ or HY_EQ with HY_NOT.
```

Comment:
`HY_EQ or HY_EQ` those are the same types. Also use enum for compare type if you want to restrict usable values.

Reply:
Here it is necessary to read it as | HY_EQ |  or | HY_EQ with HY_NOT |

Reply:
How about HY_NEQ? It would be more readable.